components/exercises/chapter-4-oscar/Alist.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render, screen, fireEvent } from '@testing-library/react';
```

Comment:
All the changes from the City List PR #25 shouldn't be in this PR

Reply:
it happened when I did npm i for the eslint dependency, I will be more careful within next PR